tests/phpunit/tests/media.php

```diff
@@ -3618,19 +3620,19 @@ public function test_wp_omit_loading_attr_threshold_filter() {
 		$this->reset_content_media_count();
 		$this->reset_omit_loading_attr_filter();
 
-		// Use the filter to alter the threshold for not lazy-loading to the first three elements.
+		// Use the filter to alter the threshold for not lazy-loading to the first five elements.
 		add_filter(
 			'wp_omit_loading_attr_threshold',
 			function() {
```

Comment:
Since `$this` is not used, this will [reduce memory](https://stackoverflow.com/a/64481894/93579) (marginally):

```suggestion
			static function() {
```

Reply:
I went ahead and used static. However, I didn't see any change in the execution time difference with and without static.

Reply:
@kt-12 Yes, this is a very minor efficiency improvement, but realistically to small to be notable in overall performance. Still a good practice to follow :)